src/components/modeler/Modeler.vue

```diff
@@ -879,9 +877,17 @@ export default {
     toXML(cb) {
       this.moddle.toXML(this.definitions, { format: true }, cb);
     },
+    onCreateElementHandler({ event, control }) {
+      // eslint-disable-next-line no-console
+      console.log('onCreateElementHandler', control);
+      this. handleDrop({ 
```

Comment:
```suggestion
      this.handleDrop({ 
```

Remove this white space